portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/*
+* Scrolls to top of element with id "top"- top of page. Adjusts scroll position to account for
+* height for header/navigation bar
+*/
+function scrollToID(id) {
+  var y = document.getElementById(id).offsetTop - document.getElementById("header").offsetHeight;
+  //console.log("height of header " + document.getElementById("header").offsetHeight);
+  console.log("top of elm " + document.getElementById(id).offsetTop);
+  //console.log("jumping to " + y);
+  window.scrollTo({top: y, behavior: 'smooth'});
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+window.onscroll = function() {
```

Comment:
You could simplify to this:

```js
window.onscroll = highlightTab;
```

Even better though, you could use the [scroll event](https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event), which would allow you to add multiple listeners and avoid overriding whatever is currently set as `window.onscroll`.